### ISSUE TITLE
Some major packages need to be updated

### ISSUE THREAD

**Issue by Ludee** [OpenEnergyPlatform/oeplatform#486]
Identified packages that need an update:

- Bootstrap 3 -> 4
- Django [1.11 (LTS: Supported until at least April 2020) ](https://www.djangoproject.com/weblog/2017/apr/04/django-111-released/)
-> [2.2 (LTS: Supported until at least April 2022) ](https://docs.djangoproject.com/en/dev/releases/2.2.3/)
-> [3.0 (UNDER DEVELOPMENT)](https://docs.djangoproject.com/en/dev/releases/3.0/)
Under test in #485 
- recline.js -> leaflet & plotly ? (this issue is further described in #491 and solution is WIP in #499)
- datatables (this is WIP in #498)

Roadmap for merging : 

1. @MGlauer + @Bachibouzouk  : #499, then #498, then #485 
2. @4lm, @johannwagner : work on boostrap 4

**Comment by MGlauer** [comment]
Tables use recline, too
- datatables

**Comment by Bachibouzouk** [comment]
> Tables use recline, too
> 
>     * datatables

I am not sure what you mean? Is it a problem statement? or a todolist?

**Comment by Bachibouzouk** [comment]
https://github.com/OpenEnergyPlatform/oeplatform/tree/feature/recline_alternatives

**Comment by MGlauer** [comment]
> > Tables use recline, too
> > ```
> > * datatables
> > ```
> 
> I am not sure what you mean? Is it a problem statement? or a todolist?

That is a continuation of @Ludee's List

**Comment by MGlauer** [comment]
Ok, I [replaced the tables](https://github.com/OpenEnergyPlatform/oeplatform/tree/feature/recline_alternatives) as @Bachibouzouk pointed out. Additionally, I also replaced the map-functionality. This whole process had some implications:

- Tables and maps are now displayed at the same time - Only the data displayed in the table is displayed in the data (thereby circumventing #369). 
- I integrated [jQuery Query Builder](https://querybuilder.js.org): This library was the only one delivering the functionality I was looking for. Unfortunately its transition to Bootstrap 4 is not done it. According to [this](https://github.com/mistic100/jQuery-QueryBuilder/issues/432), this can be worked around, tho. (Did that hopefully)
![image](https://user-images.githubusercontent.com/7835111/69442128-48e71200-0d4c-11ea-9b9f-8d87ef1dfccc.png)

I focused at functionality. The design of the functionality has do be done by those more capeable.
